LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
-Copyright (c) 2022 @BuxOrg
+Copyright (c) 2024 BSV Blockchain Association ("BA")
```

Comment:
I wonder if we shouldn't put 2023 here 🤔

Reply:
Also I agree - I think we mantained on behalf BA almost whole 2023 so I put 2023- there